tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -88,6 +88,7 @@ options:
 	nydus
 	qemu
 	qemu-tdx-experimental
+	qemu-snp-experimental
```

Comment:
@fitzthum, we're trying to keep those alphabetically sorted. Mind to move this entry before the qemu-tarball?